### PR TITLE
Fee card component

### PR DESCRIPTION
**Pull request by danjm**
This PR adds a component that can be used for displaying fees in various scenarios.

Screenshot from 2020-08-10 12-36-25
Screenshot from 2020-08-10 12-34-40

A storybook story is added for this component.

### CODE REVIEW

File: ui/app/pages/token/fee-card/fee-card.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export default function FeeCard ({
+  onFeeRowClick = null,
+  feeRowText = '',
```
Comment (whymarrh) [inline ui/app/pages/token/fee-card/fee-card.js:6]: Does this need a default? `feeRowText` is marked as required, as it should be.

File: ui/app/pages/token/fee-card/fee-card.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  feeRowLinkText = '',
+  primaryFee = '',
```
Comment (whymarrh) [inline ui/app/pages/token/fee-card/fee-card.js:8]: Does this need a default? `primaryFee` is marked as required, as it should be.

File: ui/app/pages/token/fee-card/fee-card.stories.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <FeeCard
+        onFeeRowClick={action('Fee row link clicked')}
+        feeRowText={text('feeRowText', 'Network fees')}
+        feeRowLinkText={text('feeRowLinkText', 'Edit')}
+        primaryFee={text('primaryFee', '1 ETH')}
+        secondaryFee={text('secondaryFee', '$300.57')}
+        onSecondRowClick={action('Second row link clicked')}
+        secondRowText={text('secondRowText', 'This calls a contract')}
+        secondRowLinkText={text('secondRowLinkText', 'Learn More')}
+        hideSecondRow={false}
```
Comment (whymarrh) [inline ui/app/pages/token/fee-card/fee-card.stories.js:26]: This is the default value, do we need to set it explicitly here?

```suggestion
```

File: ui/app/pages/token/fee-card/index.scss [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    margin-right: 8px;
+  }
+
+  &__row-fee {
+    margin-right: 4px;
+  }
+
+  &__icon {
+
+  }
```
Comment (whymarrh) [inline ui/app/pages/token/fee-card/index.scss:68]: This doesn't need to be here

**Review by whymarrh — CHANGES_REQUESTED** [review]
A few quick comments

**Review by jakehaugen — DISMISSED** [review]

File: .storybook/main.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../ui/app/components/**/*.stories.js'],
+  stories: ['../ui/app/**/**/*.stories.js'],
```
Comment (Gudahtt) [inline .storybook/main.js:2]: Nit: This additional `**` is redundant; `**` is already recursive.

```suggestion
  stories: ['../ui/app/**/*.stories.js'],
```

**Comment by danjm** [comment]
@whymarrh @Gudahtt all comments addressed in latest commit 22bb754a6

**Review by whymarrh — DISMISSED** [review]
LGTM

**Comment by danjm** [comment]
Just pushed fixes to lint issues in the scss file, and screenshots are the same:

Screenshot from 2020-08-10 14-41-09
Screenshot from 2020-08-10 14-41-00

**Review by whymarrh — APPROVED** [review]

**Pull request merged**